### PR TITLE
Adding iOS development pod

### PR DESCRIPTION
**Pull request by shubhankaryash**
#4

### CODE REVIEW

File: youtube-ios/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# references:
+# * http://www.objc.io/issue-6/travis-ci.html
+# * https://github.com/supermarin/xcpretty#usage
+
+osx_image: xcode7.3
```
Comment (anirudhramanan) [inline youtube-ios/.travis.yml]: there is already a .travis config file in the root directory. We can move these scripts there

File: youtube-ios/InlineYoutubeView.podspec [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  This library allows iOS developers to quickly embed YouTube videos within
+  their applications via a custom subclass called InlineYoutubeView which is modification of the original YTPlayerView by youtube.
+  This library provides:
+  * A managed WKWebView instance that loads the HTML code for the iframe player
+  * Objective-C wrapper functions for the JavaScript Player API
+  * YTPlayerViewDelegate for handling YouTube player state changes natively in
+  your Objective-C code
+                    DESC
+
+  s.homepage         = 'https://github.com/shubhankaryash/InlineYoutubeView'
```
Comment (anirudhramanan) [inline youtube-ios/InlineYoutubeView.podspec]: intended ?

File: youtube-ios/Example/InlineYoutubeView/YTViewController.m [hunk 3]
```diff
@@ -0,0 +1,71 @@
+//
```
Comment (anirudhramanan) [inline youtube-ios/Example/InlineYoutubeView/YTViewController.m:1]: we would have to add license header to the demo files also

File: youtube-ios/Example/InlineYoutubeView/YTViewController.h [hunk 4]
```diff
@@ -0,0 +1,12 @@
+//
```
Comment (anirudhramanan) [inline youtube-ios/Example/InlineYoutubeView/YTViewController.h:1]: same: license header

File: youtube-ios/Example/InlineYoutubeView/YTVideoCell.m [hunk 5]
```diff
@@ -0,0 +1,172 @@
+//
```
Comment (anirudhramanan) [inline youtube-ios/Example/InlineYoutubeView/YTVideoCell.m:1]: same: license header

File: youtube-ios/Example/InlineYoutubeView/YTVideoCell.h [hunk 6]
```diff
@@ -0,0 +1,21 @@
+//
```
Comment (anirudhramanan) [inline youtube-ios/Example/InlineYoutubeView/YTVideoCell.h:1]: same: license header

File: youtube-ios/Example/InlineYoutubeView/YTAppDelegate.m [hunk 7]
```diff
@@ -0,0 +1,46 @@
+//
```
Comment (anirudhramanan) [inline youtube-ios/Example/InlineYoutubeView/YTAppDelegate.m:1]: same: license header

File: youtube-ios/Example/InlineYoutubeView/YTAppDelegate.h [hunk 8]
```diff
@@ -0,0 +1,15 @@
+//
```
Comment (anirudhramanan) [inline youtube-ios/Example/InlineYoutubeView/YTAppDelegate.h:1]: same: license header

**Review by anirudhramanan — CHANGES_REQUESTED** [review]
Also, we would have to create a gif of the sample app, and add it to the root README.md file along with the usage instructions and pod dependency

File: youtube-ios/InlineYoutubeView.podspec (hunk 2 above)
Comment (shubhankaryash) [reply youtube-ios/InlineYoutubeView.podspec]: Changed this

File: youtube-ios/Example/InlineYoutubeView/YTViewController.m (hunk 3 above)
Comment (shubhankaryash) [reply youtube-ios/Example/InlineYoutubeView/YTViewController.m:1]: Have added //  Copyright © 2018 flipkart.com. All rights reserved. Shouldn't this suffice?

Comment (anirudhramanan) [reply youtube-ios/Example/InlineYoutubeView/YTViewController.m:1]: let's add the apache licence text also, similar to the one added in library files

Comment (shubhankaryash) [reply youtube-ios/Example/InlineYoutubeView/YTViewController.m:1]: Added the LICENSE text. Regarding the the 2nd license for flipkart, should I add another LICENSE file with a different name?

File: youtube-ios/Example/InlineYoutubeView/YTAppDelegate.h (hunk 8 above)
Comment (shubhankaryash) [reply youtube-ios/Example/InlineYoutubeView/YTAppDelegate.h:1]: Done

File: youtube-ios/Example/InlineYoutubeView/YTVideoCell.h (hunk 6 above)
Comment (shubhankaryash) [reply youtube-ios/Example/InlineYoutubeView/YTVideoCell.h:1]: Done

File: youtube-ios/Example/InlineYoutubeView/YTViewController.h (hunk 4 above)
Comment (shubhankaryash) [reply youtube-ios/Example/InlineYoutubeView/YTViewController.h:1]: Done

File: youtube-ios/Example/InlineYoutubeView/YTVideoCell.m (hunk 5 above)
Comment (shubhankaryash) [reply youtube-ios/Example/InlineYoutubeView/YTVideoCell.m:1]: Done

File: youtube-ios/Example/InlineYoutubeView/YTAppDelegate.m (hunk 7 above)
Comment (shubhankaryash) [reply youtube-ios/Example/InlineYoutubeView/YTAppDelegate.m:1]: Done

**Review by thekirankumar — APPROVED** [review]

**Review by anirudhramanan — APPROVED** [review]

**Pull request merged**